lib/mix/lib/mix/tasks/escript.build.ex

```diff
@@ -45,7 +45,7 @@ defmodule Mix.Tasks.Escript.Build do
       Defaults to app name. Set it to `nil` if no application should
       be started.
 
-    * `:embed_elixir` - if `true` embed elixir and its children apps
+    * `:embed_elixir` - if `true` embed `elixir` and its children apps
```

Comment:
@josevalim @ericmj in this situation, should it be `elixir`, Elixir, or `:elixir` ?